sink/tickle-repo/src/main/java/dk/dbc/dataio/sink/ticklerepo/MessageConsumer.java

```diff
@@ -96,64 +66,61 @@ public void handleConsumedMessage(ConsumedMessage consumedMessage) throws Invali
                     // before all data was available.)
                     Thread.sleep(5000);
                 } catch (InterruptedException e) {
-                    throw new SinkException(e);
+                    throw new RuntimeException(e);
                 }
                 result.insertItem(handleJobEnd(chunk.getItems().get(0), batch));
             } else {
-                chunk.getItems().forEach(
-                        chunkItem -> result.insertItem(handleChunkItem(chunkItem, batch)));
+                chunk.getItems().forEach(chunkItem -> result.insertItem(handleChunkItem(chunkItem, batch)));
             }
 
-            uploadChunk(result);
-        } catch (Exception any) {
-            LOGGER.error("Caught unhandled exception: " + any.getMessage());
-            metricsHandler.increment(TickleCounterMetrics.UNHANDLED_EXCEPTIONS);
-            throw any;
+            sendResultToJobStore(result);
+            transaction.commit();
```

Comment:
Lige for at være sikker (og jeg tror det er rigtigt):
Vi committer ikke transaktionen før chunken er stemplet OK i jobstore. Dvs en enhver fejl fra jobstoreconnectoren ruller databasen tilbage.

Reply:
Ja. Jeg har generelt ikke lavet om i transaktions logikken i forbindelse med migreringen... også selv det vi har måske ikke er superfedt. Men ja, vi bør nok genoverveje vores den strategi.